src/librbd/ImageCtx.cc

```diff
@@ -53,7 +53,7 @@ namespace {
 class ThreadPoolSingleton : public ThreadPool {
 public:
   explicit ThreadPoolSingleton(CephContext *cct)
-    : ThreadPool(cct, "librbd::thread_pool", "tp_librbd", 1,
+    : ThreadPool(cct, "librbd::thread_pool", "tp_librbd", 16,
```

Comment:
I am figuring this is for test purposes: you could revert commit 6c0ab75bc and temporarily re-use "rbd_op_threads" for the journal below.